app/Domain/Auth/Templates/login.blade.php

```diff
@@ -72,10 +74,11 @@
     <?php if ($tpl->get('oidcEnabled')) { ?>
         <?php $tpl->dispatchTplEvent('beforeOidcButton'); ?>
         <div class="">
-            <center class="uppercase"><?php echo $tpl->language->__("label.or"); ?></center><br />
-            <a href="{{ BASE_URL }}/oidc/login" style="width:100%;" class="btn btn-primary">
-            <?php echo $tpl->language->__("buttons.oidclogin"); ?>
-            </a>
+            <br /><center class="uppercase"><?php echo $tpl->language->__("label.or"); ?></center><br />
+            
+            <x-global::forms.button onclick="window.location.href = '{{ BASE_URL }}/oidc/login'" content-role="primary" name="oidclogin" cornerShape="full" :takeFullWidth="true">
```

Comment:
same here, lets take out corner-shape and make takeFullWidth a value for scale

Reply:
done